src/Transactions/TransactionsList/TransactionsListContainer.js

```diff
@@ -44,9 +45,11 @@ const TransactionsListContainer = ({ mutator, resources, history, match }) => {
   );
   const closePane = useCallback(
     () => {
-      history.push(`${BUDGET_ROUTE}${budgetId}${BUDGET_VIEW_ROUTE}`);
+      return match.params.id
+        ? history.push(`${FUNDS_ROUTE}/view/${match.params.id}`)
+        : history.push(`${BUDGET_ROUTE}${budgetId}${BUDGET_VIEW_ROUTE}`);
```

Comment:
what if one more place with calling list will be introduced? IMHO, it's not a solution. closePane could be passed as prop from parent container

Reply:
it stays the same, do you prefer just to skip questions in comments?